### PR TITLE
FAT-2394 C356809 Verify uploading file with Item accession number

### PR DESCRIPTION
**Pull request by MaksimTerekhov**

### CODE REVIEW

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-users-ids.spec.js [hunk 1]
```diff
@@ -14,109 +14,113 @@ const invalidUserUUID = getRandomPostfix();
...
+      BulkEditSearchPane.uploadFile(importFileName);
+      BulkEditSearchPane.waitFileUploading();
+      BulkEditActions.clickNext();
+      BulkEditActions.commitChanges();
+
+      BulkEditSearchPane.verifyChangedResults(user.username);
+      BulkEditSearchPane.verifyErrorLabelAfterChanges(userUUIDsFileName, 1, 1);
+      BulkEditActions.newBulkEdit();
+    });
+  })
```
Comment (IhorBohdan) [inline cypress/integration/bulk-edit/csv/bulk-edit-csv-users-ids.spec.js:125]: Missing semicolon

File: cypress/integration/bulk-edit/in-app/bulk-edit-in-app-items.spec.js [hunk 2]
```diff
@@ -15,9 +15,11 @@ let user;
 const item = {
```
Comment (IhorBohdan) [inline cypress/integration/bulk-edit/in-app/bulk-edit-in-app-items.spec.js:15]: fix linter issue

File: cypress/integration/bulk-edit/in-app/bulk-edit-in-app-items.spec.js [hunk 3]
```diff
@@ -30,18 +32,27 @@ describe('bulk-edit', () => {
...
         FileManager.createFile(`cypress/fixtures/${validItemBarcodesFileName}`, item.itemBarcode);
+        FileManager.createFile(`cypress/fixtures/${validItemAccessionNumbersFileName}`, item.accessionNumber);
       });
   });
 
   after('delete test data', () => {
-    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.itemBarcode);
-    Users.deleteViaApi(user.userId);
+    // InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(item.itemBarcode);
+    // Users.deleteViaApi(user.userId);
```
Comment (TetianaParanich) [inline cypress/integration/bulk-edit/in-app/bulk-edit-in-app-items.spec.js:52]: are these comments necessary?

**Review by TetianaParanich — APPROVED** [review]

File: cypress/integration/bulk-edit/csv/bulk-edit-csv-users-ids.spec.js (hunk 1 above)
Comment (MaksimTerekhov) [reply cypress/integration/bulk-edit/csv/bulk-edit-csv-users-ids.spec.js:125]: fixed

File: cypress/integration/bulk-edit/in-app/bulk-edit-in-app-items.spec.js (hunk 3 above)
Comment (MaksimTerekhov) [reply cypress/integration/bulk-edit/in-app/bulk-edit-in-app-items.spec.js:52]: fixed

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**